### DISCUSSION TITLE
技術選定

### DISCUSSION THREAD

**Discussion by t-sakurai816** [t-sakurai816/inYourFreeTime#3] [Ideas]
どのような技術を用いて構築するかを話し合います
言語、クラウド、サーバーレス等

**Comment by t-sakurai816** [comment]
---
言語 : "Node.js"
フレームワーク : ["Express", "ServerlessFramework"]
デプロイ : "AWS"
サーバーレス : "有"
---

こちらの記事等を参考にExpressで作成して、サーバーレスで公開する。＊他にも参考になる記事があるかもしれない
https://dev.classmethod.jp/articles/deploy-restapi-using-serverless-and-express/


この構成のメリットは

1. ServerlessFramework単体で作るには、SLSの使い方を覚えないといけない。（Lambdaの書き方等）
2. SLSが失敗しても最悪Expressをサーバーにデプロイすれば良い。

**Reply by t-sakurai816** [reply]
本家のチュートリアルがあったよ
https://www.serverless.com/blog/serverless-express-rest-api

**Comment by yCroma** [comment]
**言語 : "JavaScript" フレームワーク : "Vue" デプロイ : "Netlify" サーバーレス : "有"**

デザインとセキュリティを完璧にして完成を目指す。

2人で共通のフレームワークがVueだから、それを採用

Sassとlinterは入れる予定

**Reply by yCroma** [reply]
Vuetifyを利用するためにVue2を利用しまーす。
WebPackで環境を構築するかは構想を立ててるところでーす。

**Reply by yCroma** [reply]
静的配信に合わせてビルドする方法を考えていなかったので考えます。
